ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -98,14 +95,16 @@ def get_classification(self, image):
                                                       self.detected_scores_tensor,
                                                       self.detected_labels_tensor],
                                                      feed_dict={self.image_tensor: image})
-            # Filter for probability (score)
+            # Filter for probability (score) and classification
             for i, score in enumerate(scores):
-                if score > self.detection_threshold:
+                if score > self.detection_threshold and labels[i] != traffic_light_id:
```

Comment:
Or just use '4' instead of traffic_light_id? Less confusing I believe?
Ignoring even likely and large "unknown" traffic lights sounds like a good idea.